src/main/resources/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration/config.jelly

```diff
@@ -44,4 +47,5 @@
         </f:entry>
     </f:section>
 
+    <script src="${rootURL}/plugin/tm4j-automation/js/configuration/Tm4jGlobalConfiguration/index.js"/>
```

Comment:
@pelizza not sure on how is the jenkins plugin deployment/release process but i'd like to point out that now we are loading static resources like this line of code and i'm not sure if we need to do something else regarding the jar generation. Probably it's going to work fine but, because i haven't found any documentation about that, i get i bit concerned.

Reply:
I've packaged it and tested, and it seems to be working fine.